pkgs/by-name/au/audio-sharing/package.nix

```diff
@@ -59,6 +60,7 @@ stdenv.mkDerivation (finalAttrs: {
     gst_all_1.gstreamer
     gtk4
     libadwaita
+    dbus
```

Comment:
To fix build failure. Seems working on hydra though.

Reply:
This is because `wrapGAppsHook` propagating `dbus`:

```
=> nix why-depends pkgs#wrapGAppsHook pkgs#dbus.dev
/nix/store/96cwp2wv6ygbbs8wf0zr19l2yxvapm50-wrap-gapps-hook
└───/nix/store/irw5pza27z2hz0q5l0kr3r6a2a42hvhy-gtk+3-3.24.41-dev
    └───/nix/store/rhzfjivm61d5ki91kbfy4gsz8dgmizqs-at-spi2-core-2.50.0-dev
        └───/nix/store/i5yfzfynjcgb13f9gsjlyxdhrp1zsyry-dbus-1.14.10-dev

```

Tested locally, builds fine with `wrapGAppsHook` and without `dbus`, but not with `wrapGAppsHook4`.